src/main/proto/inter_node_rpc_service.proto

```diff
@@ -81,20 +81,21 @@ enum MetricEnum {
   HEAP_USAGE = 0 [(additional_fields).name = "heap usage", (additional_fields).description = "memory usage in percentage"];
   PROMOTION_RATE = 1 [(additional_fields).name = "promotion rate", (additional_fields).description = "mb/s"];
   MINOR_GC = 2 [(additional_fields).name = "minor gc", (additional_fields).description = "time in percentage"];
+  HEAP_MAX = 3 [(additional_fields).name = "heap max", (additional_fields).description = "max heap size in bytes"];
 
   // hardware
-  CPU_USAGE = 3 [(additional_fields).name = "cpu usage", (additional_fields).description = "num of cores"];
-  TOTAL_THROUGHPUT = 4 [(additional_fields).name = "total throughput", (additional_fields).description = "number of bytes read/written per second"];
-  TOTAL_SYS_CALLRATE = 5 [(additional_fields).name = "total sys callrate", (additional_fields).description = "read and write system calls per second"];
+  CPU_USAGE = 4 [(additional_fields).name = "cpu usage", (additional_fields).description = "num of cores"];
```

Comment:
Is there a reason we changed older protobuf values? Changing protobufs is a breaking change and causes compatibility issues across versions.

Reply:
Modified the values.